composer.json

```diff
@@ -8,10 +8,10 @@
     }
   ],
   "require": {
-    "php": "^8.0 || ^8.1",
+    "php": "^8.0 || ^8.1 || ^8.2",
```

Comment:
There is no need for adding `^8.2`, als `^8.0` already covers everything below `9.0`. The same is true for the already existing `^8.1` of course. ;)